<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contextual Image Mapping</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutControls" stretch="0,0,0,1,0">
        <item>
         <widget class="QPushButton" name="buttonOpenReferenceImage">
          <property name="text">
           <string>Open reference image...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonOpenMovingImages">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Open moving images...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonClearAllImages">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Clear all images</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerControls">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSaveScript">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save napari script...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableCurrentFiles">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>250</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutLandmarks" stretch="0,0">
        <item>
         <layout class="QGridLayout" name="gridLayoutReference" rowstretch="0,1,0" columnstretch="0,1,0">
          <item row="0" column="0">
           <widget class="QLabel" name="labelReference">
            <property name="text">
             <string>Reference</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="buttonClearReference">
            <property name="text">
             <string>Clear reference landmarks</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacerReference">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="MplWidget" name="mplReferenceImage" native="true">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QComboBox" name="comboSelectReference"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayoutMoving" rowstretch="0,1,0" columnstretch="0,1,0">
          <item row="0" column="0">
           <widget class="QLabel" name="labelMoving">
            <property name="text">
             <string>Moving</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="buttonClearMoving">
            <property name="text">
             <string>Clear moving landmarks</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="buttonRegister">
            <property name="text">
             <string>Register</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacerMoving">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="MplWidget" name="mplMovingImage" native="true">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QComboBox" name="comboSelectMoving"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1012</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
